src/System.Private.Reflection.Core/src/System/Reflection/Runtime/General/Dispensers.cs

```diff
@@ -62,6 +59,41 @@ internal static RuntimeAssembly GetRuntimeAssemblyFromByteArray(byte[] rawAssemb
             return result;
         }
 
+        /// <summary>
+        /// Returns non-null or throws.
+        /// </summary>
+        internal static RuntimeAssembly GetRuntimeAssemblyFromPath(string assemblyPath, string nativeImagePath)
+        {
+            byte[] rawAssembly;
+            using (var stream = new FileStream(assemblyPath, FileMode.Open, FileAccess.Read, FileShare.Read | FileShare.Delete))
+            {
+                rawAssembly = new byte[stream.Length];
+                stream.Read(rawAssembly, 0, rawAssembly.Length);
+            }
+
+            byte[] pdbSymbolStore = null;
+            if (nativeImagePath != null)
```

Comment:
`nativeImagePath` is not same as PDB symbols. `nativeImagePath` should be ignore for CoreRT.